### PR TITLE
MC-17881: add api docs for IPv4 CIDRs attached to a VPC

### PR DESCRIPTION
**Pull request by joelplourde4**
#### Changes made
- Added docs related to CIDR blocks attached to a VPC

### CODE REVIEW

File: source/includes/aws/_cidrblocks.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+### CIDRs
+
+A Classless Inter-Domain Routing (CIDR) block has to be associated to a VPC. Therefore, you need to provide the VPC ID in order to fetch the CIDR blocks
```
Comment (m-orfanos) [inline source/includes/aws/_cidrblocks.md]:
```suggestion
A Classless Inter-Domain Routing (CIDR) block has to be associated to a VPC. Therefore, you need to provide the VPC ID in order to fetch the CIDR blocks.
```

File: source/includes/aws/_cidrblocks.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-------------------- LIST CIDRs BY VPC -------------------->
+
+#### Get CIDRs by VPC
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/test-env/cidrblocks?vpcId=<vpc-id>"
```
Comment (m-orfanos) [inline source/includes/aws/_cidrblocks.md]: the param placeholders are usually displayed with a `:` 

```suggestion
   "https://cloudmc_endpoint/v1/services/aws/test-env/cidrblocks?vpcId=:vpc_id"
```

**Comment by joelplourde4** [comment]
Fuck me my rebase included changes which shouldn't have been included.

**Comment by joelplourde4** [comment]
Good Ol' turning it on and off again work for Pull Request as well hahaha

File: source/includes/aws/_cidrblocks.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+### CIDRs
```
Comment (emilyroser) [inline source/includes/aws/_cidrblocks.md]:
```suggestion
### CIDR Blocks
``` 
how do we feel abt making this swap

File: source/index.html.md [hunk 4]
```diff
@@ -233,7 +233,7 @@ includes:
   - aws/subnetworks
   - aws/internetgateways
   - aws/elastic_ips
-
+  - aws/cidrs
```
Comment (emilyroser) [inline source/index.html.md]:
```suggestion
  - aws/cidr_blocks
```

File: source/includes/aws/_cidrblocks.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+A Classless Inter-Domain Routing (CIDR) block has to be associated to a VPC. Therefore, you need to provide the VPC ID in order to fetch the CIDR blocks.
+
+<!-------------------- LIST CIDRs BY VPC -------------------->
+
+#### Get CIDRs by VPC
```
Comment (emilyroser) [inline source/includes/aws/_cidrblocks.md]:
```suggestion
#### Get CIDR Blocks by VPC
```

**Review by emilyroser — APPROVED** [review]
suggestions but approved either way!

**Review by m-orfanos — APPROVED** [review]

File: source/includes/aws/_cidr_blocks.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+### CIDRs
```
Comment (joelplourde4) [inline source/includes/aws/_cidr_blocks.md]:
```suggestion
### CIDR Blocks
```

File: source/includes/aws/_cidr_blocks.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+A Classless Inter-Domain Routing (CIDR) block has to be associated to a VPC. Therefore, you need to provide the VPC ID in order to fetch the CIDR blocks.
+
+<!-------------------- LIST CIDRs BY VPC -------------------->
```
Comment (joelplourde4) [inline source/includes/aws/_cidr_blocks.md]:
```suggestion
<!-------------------- LIST CIDR BLOCKS BY VPC -------------------->
```

File: source/includes/aws/_cidr_blocks.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Get CIDRs by VPC
```
Comment (joelplourde4) [inline source/includes/aws/_cidr_blocks.md]:
```suggestion
#### Get CIDR Blocks by VPC
```

**Pull request merged**